tools/killsnoop.bt

```diff
@@ -95,7 +97,7 @@ tracepoint:syscalls:sys_exit_tgkill
 	} else {
 		printf(" %10s", @SIG_STR[@tsig[tid]]);
 	}
-	printf(" %8d %6d\n", @tpid[tid], args.ret);
+	printf(" %8d %-16s %6d\n", @tpid[tid], comm(@tpid[tid]), args.ret);
```

Comment:
Let's hold off on adding this to the tool since it's not supported on our min kernel version.

Reply:
Indeed, thanks.